hpms/pom.xml

```diff
@@ -15,6 +15,14 @@
     </properties>
 
     <dependencies>
+        <dependency>
```

Comment:
These should be coming in from other projects I could be wrong but don't think you need them

Reply:
Fixed